python_modules/dagster/dagster/_core/definitions/repository_definition.py

```diff
@@ -850,19 +893,23 @@ def from_list(
                     schedule_def, coerced_graphs, unresolved_jobs, pipelines_or_jobs, target
                 )
 
-        # resolve all the UnresolvedAssetJobDefinitions using the full set of assets
-        for name, unresolved_job_def in unresolved_jobs.items():
-            if not combined_asset_group:
-                raise DagsterInvalidDefinitionError(
-                    f"UnresolvedAssetJobDefinition {name} specified, but no AssetsDefinitions exist"
-                    " on the repository."
+        if unresolved_jobs:
+            for (
```

Comment:
why do we need to reevaluate unresolved asset schedules again when resolving unresolved jobs?

Reply:
OOPS